frontend/app/_components/AuthComponents/LogIn.js

```diff
@@ -23,17 +22,18 @@ const LogIn = () => {
       setFormError("Please fill in both fields.");
       return;
     }
-    try {
+
       await handleLogIn(email, password, async () => {
         const userInfo = await getUserInfo();
         const userRole = userInfo.role;
         Cookies.set("userRole", userRole);
+        Cookies.set("userId", userInfo.id);
+        toast.success("Logged in successfully! 🎉");
+
         const dashboardPath = getUserPathByRole(userRole);
         window.location.replace(dashboardPath);
       });
-    } catch (error) {
-      setFormError("Login failed. Please try again.");
-    }
+
```

Comment:
why did you remove the try and catch, isnt that good for detecting errors there?

Reply:
I've handle these responds in the functions at resHandler.js. So I donot need the try catch here.